src/shims/netfxreference.props

```diff
@@ -0,0 +1,37 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <NetFxReference Include="mscorlib">
```

Comment:
Do you really need all these references?

Reply:
Once I get it building I'll check to see that we have the minimum needed.